src/components/icons.js

```diff
@@ -65,6 +65,7 @@ export const profilePic = ({ size, style, ...props } = {}) => img({
   src: getUser().imageUrl,
   height: size, width: size,
   style: { borderRadius: '100%', ...style },
+  referrerPolicy: 'no-referrer',
```

Comment:
I did this to get the profile pic to show up in local testing.

Reply:
I've skimmed through some discussions of this and I _think_ this is fine, but hopefully someone with more understanding of the referrer policy can also double-check.

Reply:
I did the same thing in TDR.  This was actually an issue even before the B2C stuff